R/generate_simulated_data.R

```diff
@@ -101,7 +101,7 @@
 #'   phi_rt = 0.6,
 #'   sigma_eps = sqrt(0.02),
 #'   scaling_factor = 0.01,
-#'   aux_site_bool = TRUE
+#'   aux_site_bool = FALSE
```

Comment:
In the example, wouldn't we want the auxiliary site to be true because we are going to use the spatial correlation function for the ww sites?